lib/dues_calculator.rb

```diff
@@ -9,16 +9,20 @@ def self.dues_per_competitor(country_iso2, base_entry_fee_lowest_denomination, c
   end
 
   def self.dues_per_competitor_in_usd(country_iso2, base_entry_fee_lowest_denomination, currency_code)
-    country_band = CountryBand.find_by(iso2: country_iso2)&.number
+    country_band = CountryBand.find_by(iso2: country_iso2)
+    country_band_detail = country_band&.active_country_band_detail
+    registration_fees = Money.new(base_entry_fee_lowest_denomination, currency_code).exchange_to("USD")
```

Comment:
The name should make clear that we're dealing with the reg fees in US dollars.

Reply:
Okay renamed.